### ISSUE TITLE
Crash: precondition failure in HTTPConnectionPool+HTTP2Connections.swift:209

### ISSUE THREAD

**Issue by nosyjoe** [swift-server/async-http-client#737]
When trying a GET request to `https://www.rtp.pt/play/podcast/252` using this code 

```swift
import AsyncHTTPClient

let request = HTTPClientRequest(url: "https://www.rtp.pt/play/podcast/252")
let response = try await HTTPClient.shared.execute(request, timeout: .seconds(30))
```

this crash happens:

```
#0	0x00000001989d9564 in _swift_runtime_on_report ()
#1	0x0000000198a9798c in _swift_stdlib_reportFatalErrorInFile ()
#2	0x00000001986860e4 in closure #1 in closure #1 in _assertionFailure(_:_:file:line:flags:) ()
#3	0x00000001986851e0 in _assertionFailure(_:_:file:line:flags:) ()
#4	0x00000001000da554 in HTTPConnectionPool.HTTP2ConnectionState.release() at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/async-http-client/Sources/AsyncHTTPClient/ConnectionPool/State Machine/HTTPConnectionPool+HTTP2Connections.swift:209
#5	0x00000001000e0528 in HTTPConnectionPool.HTTP2Connections.releaseStream(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/async-http-client/Sources/AsyncHTTPClient/ConnectionPool/State Machine/HTTPConnectionPool+HTTP2Connections.swift:644
#6	0x00000001000e9fd0 in HTTPConnectionPool.HTTP2StateMachine.http2ConnectionStreamClosed(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/async-http-client/Sources/AsyncHTTPClient/ConnectionPool/State Machine/HTTPConnectionPool+HTTP2StateMachine.swift:404
#7	0x00000001000f56f0 in closure #2 in HTTPConnectionPool.StateMachine.http2ConnectionStreamClosed(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/async-http-client/Sources/AsyncHTTPClient/ConnectionPool/State Machine/HTTPConnectionPool+StateMachine.swift:218
#8	0x00000001000f2684 in HTTPConnectionPool.StateMachine.HTTPVersionState.modify<τ_0_0>(http1:http2:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/async-http-client/Sources/AsyncHTTPClient/ConnectionPool/State Machine/HTTPConnectionPool+StateMachine.swift:88
#9	0x00000001000f55dc in HTTPConnectionPool.StateMachine.http2ConnectionStreamClosed(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/async-http-client/Sources/AsyncHTTPClient/ConnectionPool/State Machine/HTTPConnectionPool+StateMachine.swift:215
#10	0x000000010009e7dc in closure #1 in HTTPConnectionPool.http2ConnectionStreamClosed(_:availableStreams:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/async-http-client/Sources/AsyncHTTPClient/ConnectionPool/HTTPConnectionPool.swift:542
#11	0x00000001000974dc in closure #1 in HTTPConnectionPool.modifyStateAndRunActions(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/async-http-client/Sources/AsyncHTTPClient/ConnectionPool/HTTPConnectionPool.swift:212
#12	0x00000001000a51d8 in partial apply for closure #1 in HTTPConnectionPool.modifyStateAndRunActions(_:) ()
#13	0x00000001001923fc in NIOLock.withLock<τ_0_0>(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOConcurrencyHelpers/NIOLock.swift:237
#14	0x0000000100096290 in HTTPConnectionPool.modifyStateAndRunActions(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/async-http-client/Sources/AsyncHTTPClient/ConnectionPool/HTTPConnectionPool.swift:211
#15	0x000000010009e578 in HTTPConnectionPool.http2ConnectionStreamClosed(_:availableStreams:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/async-http-client/Sources/AsyncHTTPClient/ConnectionPool/HTTPConnectionPool.swift:541
#16	0x000000010009e834 in protocol witness for HTTP2ConnectionDelegate.http2ConnectionStreamClosed(_:availableStreams:) in conformance HTTPConnectionPool ()
#17	0x000000010007a3ac in HTTP2Connection.http2StreamClosed(availableStreams:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/async-http-client/Sources/AsyncHTTPClient/ConnectionPool/HTTP2/HTTP2Connection.swift:348
#18	0x000000010007a418 in protocol witness for HTTP2IdleHandlerDelegate.http2StreamClosed(availableStreams:) in conformance HTTP2Connection ()
#19	0x000000010007c37c in HTTP2IdleHandler.run(_:context:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/async-http-client/Sources/AsyncHTTPClient/ConnectionPool/HTTP2/HTTP2IdleHandler.swift:122
#20	0x000000010007cafc in HTTP2IdleHandler.userInboundEventTriggered(context:event:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/async-http-client/Sources/AsyncHTTPClient/ConnectionPool/HTTP2/HTTP2IdleHandler.swift:88
#21	0x000000010007db64 in protocol witness for _ChannelInboundHandler.userInboundEventTriggered(context:event:) in conformance HTTP2IdleHandler<τ_0_0> ()
#22	0x00000001002f17c8 in ChannelHandlerContext.invokeUserInboundEventTriggered(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1775
#23	0x00000001002f60f8 in ChannelHandlerContext.fireUserInboundEventTriggered(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1570
#24	0x00000001005b59fc in LegacyInboundStreamMultiplexer.streamClosed(event:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2ChannelHandler+InboundStreamMultiplexer.swift:173
#25	0x00000001005b58e0 in NIOHTTP2Handler.InboundStreamMultiplexer.streamClosed(event:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2ChannelHandler+InboundStreamMultiplexer.swift:77
#26	0x00000001005c58c4 in NIOHTTP2Handler.InboundStreamMultiplexer.process(event:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2ChannelHandler.swift:656
#27	0x00000001005c05c4 in NIOHTTP2Handler.processPendingUserEvents(context:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2ChannelHandler.swift:632
#28	0x00000001005c64e8 in NIOHTTP2Handler.processOutboundFrame(context:frame:promise:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2ChannelHandler.swift:778
#29	0x00000001005c10f4 in NIOHTTP2Handler.writeBufferedFrame(context:frame:promise:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2ChannelHandler.swift:689
#30	0x00000001005c0cd4 in NIOHTTP2Handler.write(context:data:promise:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2ChannelHandler.swift:452
#31	0x00000001005c1e8c in protocol witness for _ChannelOutboundHandler.write(context:data:promise:) in conformance NIOHTTP2Handler ()
#32	0x00000001002f037c in ChannelHandlerContext.invokeWrite(_:promise:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1815
#33	0x00000001002f65ac in ChannelHandlerContext.write(_:promise:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1621
#34	0x00000001002ddcd0 in _ChannelOutboundHandler.write(context:data:promise:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelHandler.swift:249
#35	0x000000010007dbf0 in protocol witness for _ChannelOutboundHandler.write(context:data:promise:) in conformance HTTP2IdleHandler<τ_0_0> ()
#36	0x00000001002f037c in ChannelHandlerContext.invokeWrite(_:promise:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1815
#37	0x00000001002f65ac in ChannelHandlerContext.write(_:promise:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1621
#38	0x000000010064bec0 in HTTP2StreamMultiplexer.write(context:data:promise:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2StreamMultiplexer.swift:75
#39	0x000000010064cef0 in protocol witness for _ChannelOutboundHandler.write(context:data:promise:) in conformance HTTP2StreamMultiplexer ()
#40	0x00000001002f0520 in ChannelHandlerContext.invokeWriteAndFlush(_:promise:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1835
#41	0x00000001002ef00c in ChannelPipeline.writeAndFlush0(_:promise:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:855
#42	0x00000001002eeec0 in ChannelPipeline.writeAndFlush(_:promise:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:767
#43	0x00000001002db5a0 in Channel.writeAndFlush(_:promise:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/Channel.swift:189
#44	0x0000000100906e10 in protocol witness for ChannelOutboundInvoker.writeAndFlush(_:promise:) in conformance NIOTSConnectionChannel ()
#45	0x00000001002dbc50 in Channel.writeAndFlush<τ_0_0>(_:promise:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/Channel.swift:243
#46	0x0000000100642620 in HTTP2StreamChannel.networkActivationReceived() at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2StreamChannel.swift:331
#47	0x000000010065f954 in MultiplexerAbstractChannel.networkActivationReceived() at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/MultiplexerAbstractChannel.swift:138
#48	0x00000001005d222c in HTTP2CommonInboundStreamMultiplexer.streamCreated(event:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2CommonInboundStreamMultiplexer.swift:146
#49	0x000000010064c470 in HTTP2StreamMultiplexer.userInboundEventTriggered(context:event:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2StreamMultiplexer.swift:103
#50	0x000000010064ce6c in protocol witness for _ChannelInboundHandler.userInboundEventTriggered(context:event:) in conformance HTTP2StreamMultiplexer ()
#51	0x00000001002f17c8 in ChannelHandlerContext.invokeUserInboundEventTriggered(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1775
#52	0x00000001002f60f8 in ChannelHandlerContext.fireUserInboundEventTriggered(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1570
#53	0x000000010007cbc4 in HTTP2IdleHandler.userInboundEventTriggered(context:event:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/async-http-client/Sources/AsyncHTTPClient/ConnectionPool/HTTP2/HTTP2IdleHandler.swift:99
#54	0x000000010007db64 in protocol witness for _ChannelInboundHandler.userInboundEventTriggered(context:event:) in conformance HTTP2IdleHandler<τ_0_0> ()
#55	0x00000001002f17c8 in ChannelHandlerContext.invokeUserInboundEventTriggered(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1775
#56	0x00000001002f60f8 in ChannelHandlerContext.fireUserInboundEventTriggered(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1570
#57	0x00000001005b57ec in LegacyInboundStreamMultiplexer.streamCreated(event:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2ChannelHandler+InboundStreamMultiplexer.swift:169
#58	0x00000001005b5670 in NIOHTTP2Handler.InboundStreamMultiplexer.streamCreated(event:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2ChannelHandler+InboundStreamMultiplexer.swift:68
#59	0x00000001005c57ec in NIOHTTP2Handler.InboundStreamMultiplexer.process(event:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2ChannelHandler.swift:650
#60	0x00000001005c05c4 in NIOHTTP2Handler.processPendingUserEvents(context:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2ChannelHandler.swift:632
#61	0x00000001005c3818 in NIOHTTP2Handler.processFrame(_:flowControlledLength:context:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2ChannelHandler.swift:601
#62	0x00000001005c07e0 in NIOHTTP2Handler.frameDecodeLoop(context:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2ChannelHandler.swift:479
#63	0x00000001005c0434 in NIOHTTP2Handler.channelRead(context:data:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-http2/Sources/NIOHTTP2/HTTP2ChannelHandler.swift:439
#64	0x00000001005c1dc4 in protocol witness for _ChannelInboundHandler.channelRead(context:data:) in conformance NIOHTTP2Handler ()
#65	0x00000001002f1340 in ChannelHandlerContext.invokeChannelRead(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1735
#66	0x00000001002f5e88 in ChannelHandlerContext.fireChannelRead(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1548
#67	0x00000001002ddf9c in _ChannelInboundHandler.channelRead(context:data:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelHandler.swift:292
#68	0x000000010013ec3c in protocol witness for _ChannelInboundHandler.channelRead(context:data:) in conformance NWWaitingHandler<τ_0_0> ()
#69	0x00000001002f1340 in ChannelHandlerContext.invokeChannelRead(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1735
#70	0x00000001002f5e88 in ChannelHandlerContext.fireChannelRead(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1548
#71	0x00000001002ddf9c in _ChannelInboundHandler.channelRead(context:data:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelHandler.swift:292
#72	0x000000010013dc74 in protocol witness for _ChannelInboundHandler.channelRead(context:data:) in conformance HTTPClient.NWErrorHandler ()
#73	0x00000001002f1340 in ChannelHandlerContext.invokeChannelRead(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:1735
#74	0x00000001002edc14 in ChannelPipeline.fireChannelRead0(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:919
#75	0x00000001002edae4 in ChannelPipeline.fireChannelRead(_:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio/Sources/NIOCore/ChannelPipeline.swift:677
#76	0x000000010093408c in StateManagedNWConnectionChannel.dataReceivedHandler(content:context:isComplete:error:) at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-transport-services/Sources/NIOTransportServices/StateManagedNWConnectionChannel.swift:409
#77	0x0000000100933c54 in implicit closure #4 in implicit closure #3 in StateManagedNWConnectionChannel.read0() at /async-http-crash-sample-fdvnroepwpjkbacttjkwxnbpltan/SourcePackages/checkouts/swift-nio-transport-services/Sources/NIOTransportServices/StateManagedNWConnectionChannel.swift:252
#78	0x000000018fe9e7c4 in closure #1 in NWConnection.receive(minimumIncompleteLength:maximumLength:completion:) ()
#79	0x000000018fea2014 in partial apply for closure #1 in NWConnection.receiveMessage(completion:) ()
#80	0x000000018fe9e58c in thunk for @escaping @callee_guaranteed (@guaranteed OS_dispatch_data?, @guaranteed OS_nw_content_context?, @unowned Bool, @guaranteed OS_nw_error?) -> () ()
#81	0x00000001020829d0 in _dispatch_block_async_invoke2 ()
#82	0x000000010206eb4c in _dispatch_client_callout ()
#83	0x0000000102077cd8 in _dispatch_lane_serial_drain ()
#84	0x0000000102078e00 in _dispatch_lane_invoke ()
#85	0x0000000102077ae8 in _dispatch_lane_serial_drain ()
#86	0x0000000102078e00 in _dispatch_lane_invoke ()
#87	0x000000010208877c in _dispatch_root_queue_drain_deferred_wlh ()
#88	0x0000000102087a54 in _dispatch_workloop_worker_thread ()
#89	0x0000000101f13d9c in _pthread_wqthread ()
Enqueued from com.apple.network.connections (Thread 4) Queue : com.apple.network.connections (serial)
#0	0x00000001020748ac in dispatch_async ()
#1	0x00000001901929e4 in nw_connection_async_client_if_needed_with_override ()
#2	0x000000019044cbf4 in nw_read_request_report ()
#3	0x000000019044f53c in nw_read_request_list_report ()
#4	0x00000001900b6a50 in nw_flow_process_input_frames(NWConcrete_nw_endpoint_handler*, NWConcrete_nw_endpoint_flow*, nw_protocol*, nw_frame_array_s*, bool, bool, bool) ()
#5	0x00000001900b3614 in nw_flow_service_reads(NWConcrete_nw_endpoint_handler*, NWConcrete_nw_endpoint_flow*, nw_flow_protocol*, bool) ()
#6	0x00000001900db604 in nw_flow_input_available(nw_protocol*, nw_protocol*) ()
#7	0x000000018df4b3a8 in nw_boringssl_read ()
#8	0x000000018df4c4b4 in nw_protocol_boringssl_input_available ()
#9	0x0000000198609720 in nw_protocol_tcp_wake_read ()
#10	0x0000000198610fcc in nw_protocol_tcp_input_flush ()
#11	0x00000001907a2bcc in nw_channel_update_input_source(nw_channel*, nw_protocol*, bool) ()
#12	0x00000001907a0324 in invocation function for block in nw_channel_create(nw_context*, unsigned char*, unsigned int, void*, unsigned int, bool, bool, bool*) ()
#13	0x000000010206eb4c in _dispatch_client_callout ()
#14	0x000000010207239c in _dispatch_continuation_pop ()
#15	0x000000010208ddcc in _dispatch_source_latch_and_call ()
#16	0x000000010208c488 in _dispatch_source_invoke ()
#17	0x000000010207a8a4 in _dispatch_workloop_invoke ()
#18	0x000000010208877c in _dispatch_root_queue_drain_deferred_wlh ()
#19	0x0000000102087a54 in _dispatch_workloop_worker_thread ()
#20	0x0000000101f13d9c in _pthread_wqthread ()
#21	0x0000000101f1bab4 in start_wqthread ()
```

Thanks for looking into this!

**Comment by Lukasa** [comment]
@fabianfett Can you take a look at this?

**Comment by gjcairo** [comment]
@nosyjoe thanks for reporting this - this should be fixed with the latest release, 1.21.1 - let us know if you still encounter issues.
